dts/bindings/mmc/st,stm32-sdmmc.yaml

```diff
@@ -0,0 +1,20 @@
+description: stm32 sdmmc disk access
+
+compatible: "st,stm32-sdmmc"
+
+include: mmc.yaml
+
+properties:
+    clocks:
+        required: true
+    label:
+        type: string
+        required: true
+    cd-gpios:
+        type: phandle-array
+        required: true
+        description: Card Detect pin
+    pwr-gpios:
+        type: phandle-array
+        required: true
+        description: Power pin
```

Comment:
What does `pwr-gpios` do? Is it something custom for a particular board? 
If so I suggest you keep it out of the sdmmc binding, I suppose most implementations won't use a power pin with SD cards.

Reply:
Yes, it is for my board. I can remove it from the driver.

Reply:
I think it is ok to make it optional: `required: false`